velox/exec/TableWriter.cpp

```diff
@@ -100,6 +100,10 @@ RowVectorPtr TableWriter::getOutput() {
   }
   finished_ = true;
 
+  // Close the data sink to flush all the buffered data and metadata to storage
```

Comment:
Do we need this? noMoreInput() already called close, no?

```
  void noMoreInput() override {
    Operator::noMoreInput();
    close();
  }
```

Reply:
Ah, you are right. That's the not the problem. Will close this.